code/bmpman/bmpman.cpp

```diff
@@ -68,8 +69,8 @@ const BM_TYPE bm_type_list[] = { BM_TYPE_DDS, BM_TYPE_TGA, BM_TYPE_PNG, BM_TYPE_
 const char *bm_ext_list[] = { ".dds", ".tga", ".png", ".jpg", ".pcx" };
 const int BM_NUM_TYPES = sizeof(bm_type_list) / sizeof(bm_type_list[0]);
 
-const BM_TYPE bm_ani_type_list[] = { BM_TYPE_EFF, BM_TYPE_ANI };
-const char *bm_ani_ext_list[] = { ".eff", ".ani" };
+const BM_TYPE bm_ani_type_list[] = { BM_TYPE_EFF, BM_TYPE_ANI, BM_TYPE_PNG };
+const char *bm_ani_ext_list[] = { ".eff", ".ani", ".png" };
```

Comment:
It would be better to use .apng instead of .png for the extension. Axem encountered an issue with CFILE blocks while loading pngs that are actually pngs(single frame), not apngs.